outreach/gsoc/2024/idea-list.md

```diff
@@ -61,6 +61,7 @@ This idea aims to replace the current parser to get rid of a host of parser rela
 
 ### Description:
 
-Wayshot allows users to perform screen capture / streaming using CPU-copyback which leads to us being bottlenecked by the CPU throughput. This also includes image transformations on the CPU.
+Currently Wayshot performs screen capturing and streaming through the CPU causing bottlenecking and severe performance degradation.
 
-With the introduction of a DMA-BUF backend, we aim to expose a client-size API for directly capturing and performing all image-transformations at the GPU level. This enables high performance streaming over applications like OBS using our custom desktop portal backend.
+Through the introduction of a DMA-BUF backend and a client-side API for performing image capturing and transformations on the GPU. 
```

Comment:
This sentence sounds incomplete and not coherent.

Reply:
You are correct, this will be changed.